thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/client/cache/CollectionSchemaCacheLoader.java

```diff
@@ -120,7 +120,7 @@ private TimeSpec fromTimeFieldSpec(TimeFieldSpec timeFieldSpec, String format) {
     TimeGranularity outputGranularity = new TimeGranularity(GRANULARITY_SIZE,
         timeFieldSpec.getOutgoingGranularitySpec().getTimeType());
     TimeSpec spec =
-        new TimeSpec(timeFieldSpec.getOutGoingTimeColumnName(), outputGranularity, format);
+        new TimeSpec(timeFieldSpec.getOutgoingTimeColumnName(), outputGranularity, format);
```

Comment:
Why was this changed back? This is breaking the build again. This was fixed correctly